.goreleaser.yml

```diff
@@ -40,3 +40,23 @@ dockers:
   - "percona/mongodb_exporter:{{.Version}}"
   dockerfile: Dockerfile
   use: buildx
+nfpms:
+  - file_name_template: "{{ .ProjectName }}-{{ .Version }}.{{ .Os }}-{{ .Arch }}"
+    homepage:  https://www.percona.com/
+    description: MongoDB exporter for Prometheus
+    maintainer: Percona
+    vendor: Percona
+    license: Apache 2.0
+    formats:
+    - deb
+    - rpm
+    bindir: /usr/bin
+    replacements:
+      amd64: 64-bit
+      darwin: macOS
```

Comment:
is this needed for deb and rpm?

Reply:
I think this naming is more beautiful...no more.